package.json

```diff
@@ -145,6 +146,7 @@
     "release": "gulp release",
     "pretest": "npm run lint",
     "test": "mocha",
+    "test-e2e": "node test/e2e/server.js --config ./test/e2e/nightwatch.json",
```

Comment:
this could be prepended with `java -jar bin/selenium-server-standalone-2.51.0.jar`